### ISSUE TITLE
Indent content between cfscript tags

### ISSUE THREAD

**Issue by bhartsfield** [jcberquist/commandbox-cfformat#142]
I did not find anything in existing settings, please let me know if I am just missing it. I'd like cfformat to not line up the content in cfscript blocks with the actual cfscript opening and closing tags. Instead, I'd prefer the content between cfscript tags to be indented. Basically, everything between cfscript tags should indent by the current indent settings + 1

**Comment by jcberquist** [comment]
You are right that there isn't a setting to control this. It is currently hardcoded here: [models/cftags/CFTagBody.cfc#L35](https://github.com/jcberquist/commandbox-cfformat/blob/master/models/cftags/CFTagBody.cfc#L35). I am not opposed to adding a setting for it, however.

**Comment by onlyleura** [comment]
Here's an UP vote for this one!

**Comment by andmayfi92** [comment]
Late to the party but I would love this setting as well
